src/orion/benchmark/task/forrester.py

```diff
@@ -0,0 +1,34 @@
+"""Forrester Task from the Profet paper.
+
+This Forrester class is based on a synthetic function, whereas the ForresterTask is baseed on a
+meta-model trained on multiple such functions.
+
+Klein, Aaron, Zhenwen Dai, Frank Hutter, Neil Lawrence, and Javier Gonzalez. "Meta-surrogate
+benchmarking for hyperparameter optimization." Advances in Neural Information Processing Systems 32
+(2019): 6270-6280.
+"""
+from orion.benchmark.task.base import BenchmarkTask
+from typing import List, Dict
+import numpy as np
+
+
+class Forrester(BenchmarkTask):
+    """ Task based on the Forrester function, as described in https://arxiv.org/abs/1905.12982
+    
+    $ f(x) = ((\\alpha x - 2)^2) sin(\\beta x - 4) $
```

Comment:
max_trials, alpha and beta parameters should be documented here.

Reply:
Oops, text is over-indented.

Reply:
Fixed in 9f61fa837bc657cfaa7f9c9a3d9121b7e3402317

Reply:
Thanks!